app/src/main/java/com/github/sdp/ratemyepfl/activity/AddReviewActivity.kt

```diff
@@ -1,57 +1,69 @@
-package com.github.sdp.ratemyepfl.activity.classrooms
+package com.github.sdp.ratemyepfl.activity
 
 import android.app.Activity
 import android.content.Intent
 import android.os.Bundle
+import android.text.Editable
+import android.text.TextWatcher
 import android.widget.Button
 import android.widget.RatingBar
 import android.widget.TextView
 import androidx.activity.viewModels
 import androidx.appcompat.app.AppCompatActivity
 import com.github.sdp.ratemyepfl.R
-import com.github.sdp.ratemyepfl.activity.course.CourseReviewActivity
 import com.github.sdp.ratemyepfl.model.review.ReviewRating
-import com.github.sdp.ratemyepfl.viewmodel.AddRoomReviewViewModel
+import com.github.sdp.ratemyepfl.viewmodel.AddReviewViewModel
 import com.google.android.material.textfield.TextInputEditText
 import dagger.hilt.android.AndroidEntryPoint
 
 const val ROOM_GRADE = "grade"
 const val ROOM_COMMENT = "comment"
 
 @AndroidEntryPoint
-class AddRoomReviewActivity : AppCompatActivity() {
+class AddReviewActivity : AppCompatActivity() {
```

Comment:
Global note on the UI for Review: I think we should add a box for the title of the review. Maybe a task for the next sprint.